backends-common/postgres/src/main/java/org/apache/james/backends/postgres/PostgresTableManager.java

```diff
@@ -53,6 +55,26 @@ public Mono<Void> initializeTables() {
                 .then());
     }
 
+    private Mono<Void> alterTableEnableRLSIfNeed(PostgresTable table) {
+        if (table.isEnableRowLevelSecurity()) {
+            return alterTableEnableRLS(table);
+        }
+        return Mono.empty();
+    }
+
+    public Mono<Void> alterTableEnableRLS(PostgresTable table) {
+        return postgresExecutor.connection()
+            .flatMapMany(con -> con.createStatement(getAlterRLSStatement(table.getName())).execute())
+            .flatMap(Result::getRowsUpdated)
+            .then();
+    }
+
+    private String getAlterRLSStatement(String tableName) {
+        return "ALTER TABLE " + tableName + " ADD DOMAIN varchar(255) not null DEFAULT current_setting('app.current_domain')::text;" +
```

Comment:
I can't find the API for doing that with Jooq, 
So I try to native SQL

Reply:
I guess this should be done in https://github.com/linagora/james-project/issues/4898 instead.

Reply:
I did cherrypick this commit to postgres table manager PR